config/environments/production.rb

```diff
@@ -32,7 +32,8 @@
       user_name:            GlobalSetting.smtp_user_name,
       password:             GlobalSetting.smtp_password,
       authentication:       GlobalSetting.smtp_authentication,
-      enable_starttls_auto: GlobalSetting.smtp_enable_start_tls
+      enable_starttls_auto: GlobalSetting.smtp_enable_start_tls,
+      ssl:                  GlobalSetting.smtp_enable_ssl
```

Comment:
@tgxworld I think the below line indicates the option that I included.
- https://github.com/rails/rails/blob/master/actionmailer/lib/action_mailer/base.rb#L409

While I am not familiar with ruby & rails, I am trying to find source codes that contains the option.